curriculum/challenges/english/08-coding-interview-prep/rosetta-code/knight's-tour.md

```diff
@@ -0,0 +1,141 @@
+---
+id: 5a23c84252665b21eecc7ed5
+title: Knight's tour
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+<a href="https://en.wikipedia.org/wiki/Knight%27s_tour">Problem</a>: you have a <code>w</code> x <code>h</code> chessboard, empty but for a single knight on some square. The knight must perform a sequence of legal moves that result in the knight visiting every square on the chessboard exactly once. Note that it is <i>not</i> a requirement that the tour be "closed"; that is, the knight need not end within a single move of its start position.
```

Comment:
```suggestion
<a href="https://en.wikipedia.org/wiki/Knight%27s_tour">Knight's Tour</a>Problem: You have an empty <code>w</code> * <code>h</code> chessboard, but for a single knight on some square. The knight must perform a sequence of legal moves that result in the knight visiting every square on the chessboard exactly once. Note that it is <i>not</i> a requirement that the tour be "closed"; that is, the knight need not end within a single move of its start position.
```
This looks better to me

Reply:
This looks good! Could you make this change, @NitronR ?

Reply:
Sure.